README.md

```diff
@@ -1,26 +1,85 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+let num = 20
+while (num >= 0) {
+  console.log(num)
+  num --
+}
+
+for (num = 20; num >= 1; num -= 1){
+  console.log (num)
+}
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+let num = 20
+while (num <=20 && num >= 0) {
+if (num === 0){
+  console.log (num +' is zero')
+  }
+  if (num %2 === 0 && num !== 0){
+    console.log (num +' is even')
+  }
+  else if (num %2 >= 1){
+    console.log (num + ' is odd')
+  }
+  num --
+}
+
+
+
+for (num = 0; num <= 20; num ++){
+  if(num === 0){console.log (num +' is zero')}
```

Comment:
Watch out for syntax here! If stuff is all on one line it's easier to miss mistakes and typos. Take a look at [this guide](https://www.w3schools.com/js/js_conventions.asp) for some good info.